src/app/campaign-stats.model.ts

```diff
@@ -0,0 +1,12 @@
+export class CampaignStats {
+    constructor(
+        /**
+         * @param: total amount raised (including adjustment) for all except master campaigns
```

Comment:
I only did half a physics degree, the main thing I kept from it was that a number without a unit is meaningless. Is this right?

```suggestion
         * @param: total amount raised (including adjustment) for all except master campaigns in whole pounds GBP
```

Reply:
Also not sure what 'including adjustment' means here, no if we need to say excluding master campaigns. We only need to document the meaning of this field, not necessarily the method used to calculate it.

Reply:
Whole pounds, not sure – I don't think we really need to round it in Apex.

GBP, yes – as discussed on the SF counterpart, it should do an approx conversion to our "home" currency. I think for this endpoint, given we have no non-GBP real fundraising happening yet, it's good enough.

I also have a slight preference for just calling it `totalRaised` in the API.